lib/lexer.mll

```diff
@@ -18,7 +18,8 @@ rule text section = parse
       { let section = (String.length n, str) in
         newline lexbuf;
         `Section section :: text (Some section) lexbuf }
-  | "```" ([^' ' '\n']* as h) ws* ([^'\n']* as l) eol
+  | ( "<!--" ws* "$MDX" ws* ([^' ' '\n']* as label_cmt) ws* "-->" ws* eol )?
+      "```" ([^' ' '\n']* as h) ws* ([^'\n']* as l) eol
```

Comment:
Should we allow for one or more blank lines between the labels and the comment ?

Reply:
Good idea, done.

Reply:
Is there a good reason to allow for blank lines? If not I think we should enforce sticking the labels above the block.

If we don't enforce that we have to remember what lives between the labels and the block to reprint it as it was.

Reply:
That's a good point, I reverted the change, now we can have 0 or 1 linebreak (so no blank lines)

Reply:
Will 0 line break work? I'm guessing it should but that really depends on how external markdown processing tools work. There's no arm in allowing it anyway but we should document the following form:
```
<!-- $MDX <labels> -->
``<header>
<code>
``
```

So that we can forbid it in the future if it turns out to be an issue!

Reply:
I updated the doc, I also added a precision about labels being separated by a comma, which should be enough to close #135 if you agree

Reply:
Yep, let's close #135.

That doesn't mean we won't have to start writing proper documentation at some point though!